beacon-chain/operations/service.go

```diff
@@ -249,10 +185,6 @@ func (s *Service) HandleAttestation(ctx context.Context, message proto.Message)
 		}
 	}
 
-	if err := blocks.VerifyAttestation(bState, attestation); err != nil {
```

Comment:
we dont verify attestations in sync service 
```go
func (r *RegularSync) beaconAttestationSubscriber(ctx context.Context, msg proto.Message) error {
	if err := r.operations.HandleAttestation(ctx, msg.(*ethpb.Attestation)); err != nil {
		return err
	}

	return r.chain.ReceiveAttestationNoPubsub(ctx, msg.(*ethpb.Attestation))
}
```
we currently send them to operations service and then blockchain service

Reply:
Yea we should verify the signature. It's a todo

Reply:
Does this PR remove signature verification for attestations?

Reply:
Yes. It removes signature verification in operation service level, since it was suppose to be handled in the sync level (before gossiping) to rest of peers.

We can hold on to merge this PR til sync validates attestation

Reply:
is it a requirement to validate every attestation that comes through sync ? this could be a griefing vector. With 100 attestations that would take roughly 2s. This would be fine if all the received attestations were aggregated, but the majority we receive right now are single attestations

Reply:
Nope. The network spec has it as a todo. I agree we should keep such verification as an empty until the network spec for sync verification strategy finalizes

Reply:
This is ready then : )